Makefile

```diff
@@ -168,7 +168,7 @@ $(BUILD_DIR) $(DATA_DIR) $(DIST_DIR) $(DIST_DATA_DIR):
 	$(MKDIR) $@
 
 test: $(GIT_TEST_DATA_REPO_PATH)
-	$(TESTRUNNER) tests
+	$(TESTRUNNER) tests/test_spec_utils.py
```

Comment:
It looks like this is leftover from development and not a change we want, right?

Reply:
argh, sorry! Fixed.